src/Components/MainSection/ErrorPage.jsx

```diff
@@ -0,0 +1,11 @@
+import './ErrorPage.css';
+
+function ErrorPage() {
```

Comment:
would suggest updating the name of the file since this is not exactly an error page but an under-construction page